apps/e2e/angular-tree-component-storybook/src/app/angular-tree-component/angular-tree-component.component.html

```diff
@@ -62,14 +62,13 @@ <h3>Basic with toolbar, context menu and inline help</h3>
         #treeNodeFullTemplate
       >
         <sky-angular-tree-node
+          [helpPopoverContent]="
+            node.data.showHelp ? 'Example help inline content.' : undefined
+          "
```

Comment:
instead of replacing the old help inline with the new component, can we have two nodes with help inline (one old, one new) so we dont lose visual data on the old component